payments-core/src/test/java/com/stripe/android/payments/core/authentication/WebIntentNextActionHandlerTest.kt

```diff
@@ -200,6 +231,7 @@ class WebIntentAuthenticatorTest {
         expectedAnalyticsEvent?.let {
             verifyAnalytics(it)
         }
+        redirectResolver.turbine.ensureAllEventsConsumed()
```

Comment:
This pattern provides stronger guarantees of what's actually happening (ie, all the existing URLs didn't start triggering an SDK attempted redirect lookup).